tests/x11/libreoffice/libreoffice_pyuno_bridge_no_evolution_dep.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Case 1503978 - LibreOffice: pyuno bridge
+# Maintainer: Grace Wang <gwang@suse.com>
```

Comment:
I recommend to put a full URL to the test case within the line "# Tags: ". It is unclear to many what "Case <nr>" means and how they would be able to lookup that test description

Reply:
Thanks Oliver, I've updated the test case based on the comment.